code/datums/wires/wires.dm

```diff
@@ -120,15 +121,15 @@ var/list/wireColours = list("red", "blue", "green", "black", "orange", "brown",
 				if(iswirecutter(I))
 					var/colour = href_list["cut"]
 					CutWireColour(colour)
-					log_attack("[key_name(usr)] has [IsColourCut(colour) ? "cut" : "mended"] the [GetWireName(wires[colour])] wire of airlock named \"[holder]\" at [holder.x],[holder.y],[holder.z].")
+					holder.investigation_log(I_WIRES, "|| [GetWireName(wires[colour]) || colour] wire [IsColourCut(colour) ? "cut" : "mended"] by [key_name(usr)] ([src.type])")
```

Comment:
I believe you're missing a starting [ and you might want to remove that first ||